itest/lnd_psbt_test.go

```diff
@@ -1592,3 +1592,310 @@ func testPsbtChanFundingFailFlow(ht *lntest.HarnessTest) {
 	// funding workflow with an internal error.
 	ht.ReceiveOpenChannelError(chanUpdates, chanfunding.ErrRemoteCanceled)
 }
+
+// testPsbtChanFundingWithUnstableUtxos tests that channel openings with
+// unstable utxos, in this case in particular unconfirmed utxos still in use by
+// the sweeper subsystem, are not considered whene opening a channel. They bear
+// the risk of being RBFed and are therefore not safe to open a channel with.
+func testPsbtChanFundingWithUnstableUtxos(ht *lntest.HarnessTest) {
+	if ht.IsNeutrinoBackend() {
+		ht.Skipf("skipping test for neutrino backend because it " +
+			"does not support unconfirmed transactions")
+	}
+
+	fundingAmt := btcutil.Amount(2_000_000)
+
+	// First, we'll create two new nodes that we'll use to open channel
+	// between for this test.
+	carol := ht.NewNode("carol", nil)
+	dave := ht.NewNode("dave", nil)
+	ht.EnsureConnected(carol, dave)
+
+	// Fund Carol's wallet with an unconfirmed utxo.
+	ht.FundCoinsUnconfirmed(fundingAmt, carol)
+
+	ht.AssertNumUTXOsUnconfirmed(carol, 1)
```

Comment:
again think the test can be simplified by using the `BumpFee` - we then no longer need to create FC to generate a sweeper-used utxo and can exit the test earlier.

Reply:
see comment above, because we are opening channels anyway in this test to prove that one can open channels with unconfirmed utxos, we can just FC the channel and immediately have a sweeper related utxo for further testing.